templates/cycle.html

```diff
@@ -8,6 +8,15 @@
 {{ $votingEnabled := .VotingEnabled }}
 {{ $votesAvailable := .AvailableVotes }}
 
+{{if .LastCycle}}
+<div class="cycleHistory">
+	In the last Movienight on {{.LastCycle.EndedString}} we watched:
+	{{range .LastCycle.Watched}}
+	</br>* <a href="/movie/{{.Id}}">{{.Name}}</a>
```

Comment:
Use a `<ul>` here instead of manually making bullet points.

Reply:
fixed in 2784fe1